packages/contentful-core/package.json

```diff
@@ -43,9 +39,7 @@
     "graphql": ">=15"
   },
   "dependencies": {
-    "apollo-link-contentful": "^0.1.3",
-    "cross-fetch": "^3.1.4",
-    "invariant": "^2.2.4"
+    "cross-fetch": "^3.1.4"
```

Comment:
is this need it?, can it be replace with _global_ fetch?

Reply:
(commenting in here as I cannot comment below) are dotenv and graphql devDependencies needed?

Reply:
I've removed `cross-fetch` in favor of default Fetch API used in `HttpLink` when available.
`dontenv` has been removed too

Reply:
Regarding `graphql`:

> For version 3.x, graphql is a peer dependency of @apollo/client. This means you should install it separately in your project. The Apollo team made this change to allow developers more control over the version of graphql used in their projects.